Contributing/How-to-Coach-New-Contributors.md

```diff
@@ -1,19 +1,23 @@
-    * Ensure you have already [claimed credit](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Get-Credit-for-Your-Contribution.md) for your work in the past 
-    * Add your name to Ambassadors in the README document by copying your avatar and name from the *all-contributors* section and pasting it under the *Ambassadors* header
-    * Submit a Pull-Request with your proposed changes
+## Coach or Train new contributors
 
+moja global is increasingly looking for new contributors who can contribute to our projects and processes. To help them better involve with the community and ensure that their potential is utilized appropriately, we request our community members to coach or train new contributors. If you have experience and want to help, please send an email to [mentorship@moja.global](mailto:mentorship@moja.global).
 
-**Coaches** are experienced coders or scientists or users who are available to work shoulder to shoulder with new coders, users, or contributors   
+As a Coach or an Ambassador, you need to ensure that the new contributors get the required mentorship to help them succeed in the community and drive a positive impact.
 
-**Ambassadors** are experienced coders or scientists or users who are available to provide training to groups of new coders, users, or contributors 
+1.  Ensure that the Contributors know that you are a Coach.
 
+    -   Ensure that you have a track record that proves you know the repository, the processes, the purpose and the contents well. 
+    -   Ensure you have already [claimed credit](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Get-Credit-for-Your-Contribution.md) for your work in the past.
+    -   Add your name to Coaches in the `README` file by copying your avatar and name from the *all-contributors* section and pasting it under the *Coaches* header.
+    -   Submit a Pull-Request with your proposed changes. It would be reviewed and merged.
+
+2.  Ensure that the Contributors know that you are an Ambassador.
+    -   Ensure that you have a track record that proves you know the repository, the processes, the purpose and contents well.
+    -   Ensure that you have gained credit as a Coach and that you have experience in delivering training to new contributors.
+    -   Ensure you have already [claimed credit](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Get-Credit-for-Your-Contribution.md) for your work in the past. 
+    -   Add your name to Ambassadors in the `README` document by copying your avatar and name from the *all-contributors* section and pasting it under the *Ambassadors* header.
+    -   Submit a Pull-Request with your proposed changes. It would be reviewed and merged.
+
+**Coaches** are experienced engineers or scientists or users who are available to work shoulder to shoulder with new developers, users, or contributors.
```

Comment:
Can these Coaches and Ambassadors definitions move before point 1, 2 ?
So that reader will first know who are Coaches/Ambassadors and then read pt 1,2 to understand more what needs to be done by Coaches/Ambassadors.

Reply:
Done!